data_processing.py

```diff
@@ -5,92 +5,185 @@
+            in option 2, user can select which column to select as use and parking spaces
+            (in automatic processing, the two left-most is assumed)
+
+    Args:
+        parking_table: DataFrame of extract parking table only
+        filename: {region}_{state}
+        region: region without underscores like in filename
+        state: state without underscores like in filename
+
+    Returns: None
+
+    """
     user = ""
     while user != "insert":
-        print(f"{parking_table}\nn: {parking_table.shape}\nColumn names: {parking_table.columns}\n")
+        print(f"{parking_table}\nn: {parking_table.shape}\nColumn/Header names: {parking_table.columns}\n")
         user = input(
             """Processing steps:
-                1. Remove extra headers
-                2. Remove extra columns and removes categories 
-                3. Rename columns to "Use" and "Number of Spaces" (only use with 2 columns)
-                4. Remove duplicates
-                5. Save as .csv
+                1. Rename headers
+                2. Remove extra columns and removes categories (also renames columns)
+                3. Remove duplicates (human error in the parking tables)
+                4. Save as .csv
                 Enter "insert" to start insertion.
                 Enter "exit" to exit.
                 >> """)
 
         if user == "1":
-            parking_table = remove_extra_header(parking_table)
+            parking_table = rename_headers(parking_table)
         elif user == "2":
             use = input("Enter column name to keep for...\nUse: ")
             spaces = input("Number of Spaces: ")
+
             # returns only use and spaces columns, replaces empty string with NaN, removes any rows with NaN
             parking_table = parking_table[[use, spaces]].replace(r'^s*$', float('NaN'), regex=True).dropna(axis=0)
 
-            # removes rows that have the same name as headers
+            # removes rows that have the same name as headers (most common in multi-page tables in pdfs)
             parking_table = parking_table[parking_table[use] != use]
             parking_table = parking_table[parking_table[spaces] != spaces]
-        elif user == "3":
             parking_table.columns = ["Use", "Number of Spaces"]
-        elif user == "4":
+        elif user == "3":
             parking_table = parking_table.drop_duplicates(subset="Use")
-        elif user == "5":
-            parking_table.to_csv(os.path.join('input_csv', f'{region_file}_{state}.csv'), index=False)
+        elif user == "4":
+            parking_table.to_csv(os.path.join('input_csv', f'{filename}.csv'), index=False)
         elif user == "exit":
             sys.exit("Exiting program.")
 
     insert_df(parking_table, state, region)
 
 
-def remove_extra_header(df):
-    col = len(df.columns) - 1
-    df.columns = [headers[col] for headers in list(df)]
-    return df
+def rename_headers(df):
+    """Renames DataFrame headers to its index
```

Comment:
It'd be good to rename the function to be more obvious what it does, like `rename_headers_to_index`. "self-documenting code"